Content.Server/_RMC14/Marines/ControlComputer/MarineControlComputerSystem.cs

```diff
@@ -1,5 +1,43 @@
-﻿using Content.Shared._RMC14.Marines.ControlComputer;
+﻿using Content.Server.Body.Components;
+using Content.Shared._RMC14.Commendations;
+using Content.Shared._RMC14.Marines.ControlComputer;
+using Content.Shared._RMC14.Marines;
+using Content.Shared._RMC14.Survivor;
 
 namespace Content.Server._RMC14.Marines.ControlComputer;
 
-public sealed class MarineControlComputerSystem : SharedMarineControlComputerSystem;
+public sealed class MarineControlComputerSystem : SharedMarineControlComputerSystem
+{
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<MarineComponent, BeingGibbedEvent>(OnMarineGibbed);
+    }
+
+    private void OnMarineGibbed(EntityUid uid, MarineComponent component, ref BeingGibbedEvent ev)
```

Comment:
This should be using a single Entity<MarineComponent> argument instead of the EntityUid and a component separately